src/sankoff_handler.cpp

```diff
@@ -104,14 +99,13 @@ void SankoffHandler::RunSankoff(Node::NodePtr topology) {
       for (size_t pattern_idx = 0; pattern_idx < site_pattern_.PatternCount();
            pattern_idx++) {
         auto partials_from_parent =
-            ParentPartial(psv_handler_.GetPV(PSVType::Q, node->Id())
-                              .block<state_count_, 1>(0, pattern_idx));
+            ParentPartial(psv_handler_.GetPV(PSVType::Q, node->Id()).col(pattern_idx));
         for (size_t child = 0; child < 2; child++) {
           size_t sister_idx = child == 0 ? 1 : 0;
           auto partials_from_sister = ParentPartial(
               TotalPPartial(node->Children()[sister_idx]->Id(), pattern_idx));
           psv_handler_.GetPV(PSVType::Q, node->Children()[child]->Id())
-              .block<state_count_, 1>(0, pattern_idx) =
+	      .col(pattern_idx) =
```

Comment:
This looks like a tab but we're using spaces here.

Reply:
resolved